src/arch/simddetect.cpp

```diff
@@ -160,6 +162,11 @@ SIMDDetect::SIMDDetect() {
 #endif
 #endif
 
+#if defined(NEON)
+  /* No runtime detection for NEON yet */
+  neon_available_ = 1;
```

Comment:
So neon will be enabled for all ARM systems, and those without neon will crash at runtime?

Reply:
Yes. I can't find an easy way to detect it at runtime. Possibly we need a configure option to enable/disable it.

Reply:
https://stackoverflow.com/questions/26701262/how-to-check-the-existence-of-neon-on-arm